soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -4745,9 +4741,90 @@ void DrawRandoEditor(bool& open) {
     ImGui::End();
     }*/
 
+typedef struct {
+    GetItemID giid;
+    ItemID iid;
+    std::string english;
+    std::string german;
+    std::string french;
+} GetItemMessage;
+
+void CreateGetItemMessages(std::vector<GetItemMessage> messageEntries) {
+    CustomMessageManager* customMessage = CustomMessageManager::Instance;
```

Comment:
maybe rename this to customMessageManager or something? having it just named "customMessage" feels like it should be the actual custom message